src/coreclr/jit/hwintrinsicarm64.cpp

```diff
@@ -1738,16 +1740,29 @@ GenTree* Compiler::impSpecialIntrinsic(NamedIntrinsic        intrinsic,
             break;
         }
 
+        case NI_AdvSimd_StoreSelectedScalar64x2:
```

Comment:
nit: rearrange to group the similar ones next to each other.

```c++
        case NI_AdvSimd_StoreVector64x2:
        case NI_AdvSimd_Arm64_StoreVector128x2:
        case NI_AdvSimd_StoreSelectedScalar64x2:
        case NI_AdvSimd_Arm64_StoreSelectedScalar128x2:
```

Reply:
Done